CHANGELOG.md

```diff
@@ -1,10 +1,37 @@
 # Change Log
 
+## 0.7.0 - 2024-04-11
+
+This update comes with some major changes to the `Deploy Project from Workspace...` family of commands. We have now expanded support for monorepos by enabling the deployment and saving of multiple app configurations within the same workspace project.  New settings configurations can now be found under `containerApps.deploymentConfigurations`.  Any older settings will be migrated automatically to the new schema.
+
+We have also included a new getting started walkthrough which steps through first time deployment using the Azure Container Apps extension.
+<br/>
+
+For more information please consult our [README](https://github.com/microsoft/vscode-azurecontainerapps#readme).
+
+### Added
```

Comment:
I think that this changelog is a little verbose. There are quite a few notes that are more like implementation details of a feature that it makes it difficult for a user to identify the new features of the release.

Personally, I would only include the new features in the added. I think that would be condensed to:
* [[59]](https://github.com/microsoft/vscode-azurecontainerapps/issues/39) Add a getting started walkthrough
* [[607]](https://github.com/microsoft/vscode-azurecontainerapps/issues/607) Expand support for monorepos and deploying multiple app configurations within the same workspace project


This could go into changes:
* [[634]](https://github.com/microsoft/vscode-azurecontainerapps/pull/634) Remove auto-naming of resources based on the workspace name and always prompts for new app names
* [[638]](https://github.com/microsoft/vscode-azurecontainerapps/pull/638) Leverage existing ACRs automatically if previous apps exist in saved deployment settings
* [[641]](https://github.com/microsoft/vscode-azurecontainerapps/pull/641) Add support for taking a saved `envPath` and converting it to environment variables
* [[669]](https://github.com/microsoft/vscode-azurecontainerapps/pull/669) Find previously deployed to managed environments and recommend them

The rest could be either in engineering or not included at all. Most of them are more like "implementation" details than actual changes/feature work.